pkgs/development/ocaml-modules/kicadsch/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+, ocaml
+}:
+
+assert (lib.versionAtLeast ocaml.version "4.09");
+
+buildDunePackage rec {
+  pname = "kicadsch";
+  version = "0.9.0";
+
+  src = fetchFromGitHub {
+    owner = "jnavila";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-6uz8+wDjaqxJ5DCTQZsIsN1wgzsbvPAZrVOec+47M6M=";
+  };
+
```

```suggestion
      minimalOCamlVersion = "4.07";
```

https://github.com/jnavila/plotkicadsch/blob/master/kicadsch.opam#L22